### ISSUE TITLE
Useless warning "Unable to fetch package info for orgname.gameident.editor: invalid ident"

### ISSUE THREAD

**Issue by trundlr** [Facepunch/sbox-issues#7065]
### Describe the bug

This warning does not mean anything, and it would be nice if it could be suppressed:

![Image](https://github.com/user-attachments/assets/52faf9c5-ae0a-4ce9-8f79-7aacdcdf9efe)

### To Reproduce

.

### Expected behavior

No warning

**Comment by TrvsF** [comment]
i am having issues with this

**Comment by kira0x1** [comment]
yeah this is really annoying when working on editor projects. just spams my console

**Comment by badandbest** [comment]
As the editor library guy I get this every single hotload

**Comment by kira0x1** [comment]
> As the editor library guy I get this every single hotload

honestly makes me dread making tools which i usually love doing.

**Comment by aylaylay** [comment]
Fixed

**Comment by trundlr** [comment]
Thank you so much
